<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedViews">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageFav">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="PlayListGalleryView" name="playListGallery"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageGallery">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="ImageGalleryView" name="gallery"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageImageView">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="MainGraphicsView" name="graphicsView">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::ScrollHandDrag</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="sidebar">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>289</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>PlayList</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="sidebarContent">
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QWidget" name="infoWidget" native="true">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblNamePrefix">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblWidthPrefix">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblWidth">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblHeightPrefix">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lblHeight">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblName">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="playListWidget">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ui/MainGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>ImageGalleryView</class>
   <extends>QGraphicsView</extends>
   <header>ui/ImageGalleryView.h</header>
  </customwidget>
  <customwidget>
   <class>PlayListGalleryView</class>
   <extends>QGraphicsView</extends>
   <header>ui/PlayListGalleryView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
